### PR TITLE
Fix css for disabled form-select divs.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Also removes some obsolete CSS.

### CODE REVIEW

File: packages/devtools/web/styles.css [hunk 1]
```diff
@@ -858,6 +823,12 @@ div.tabnav-tab {
     height: 30px;
 }
 
+.form-select[disabled], .form-select[disabled]:hover {
+    color: rgba(36,41,46,0.4);
+    background-color: #eff3f6;
```
Comment (terrylucas) [inline packages/devtools/web/styles.css]: should the colors be dependent on dark or light?

Comment (kenzieschmoll) [reply packages/devtools/web/styles.css]: for dark mode we just use the disabled button style for the disabled form select. If you look at styles_dark.css, I tacked `.form-select[disabled]` onto that style.

Comment (kenzieschmoll) [reply packages/devtools/web/styles.css]: For the `removed var` commit,  I accidentally left an unused var in my initial commit, so it was just a cleanup.

Comment (terrylucas) [reply packages/devtools/web/styles.css]: Thanks makes sense.

**Pull request merged**